theme/components/toolbar.scss

```diff
@@ -16,12 +16,16 @@
 		vertical-align: middle;
 		display: inline-block;
 		background: ck-border-color();
-		margin: 0 ck-spacing( 'tiny' );
 	}
 
 	&-newline {
 		display: block;
 		clear: left;
 		height: ck-spacing( 'small' );
 	}
+
+	// (#11) Separate toolbar items.
+	& > * + * {
+		margin-left: ck-spacing( 'small' );
```

Comment:
Won't that look bad if items exceeds the available width? The one that appeared in a new line will have left margin.